go-sdk/pkg/registryclient-v2/admin/admin_request_builder.go

```diff
@@ -10,13 +10,11 @@ type AdminRequestBuilder struct {
 }
 
 // ArtifactTypes the list of artifact types supported by this instance of Registry.
-// returns a *ArtifactTypesRequestBuilder when successful
```

Comment:
this diff is really strange, is something going wrong with Kiota versions?
I just regenerated Go a couple of days ago.

Reply:
I dunno - I opened the PR to see what the diff was and didn't have a chance to look right away.  But now looking at the diff, it's all just comments!  wtf?  Environmental differences?

Reply:
> Environmental differences?

Shouldn't ...

Reply:
:shrug: